src/test/java/org/fundacionjala/coding/RentalTest.java

```diff
@@ -0,0 +1,43 @@
+package org.fundacionjala.coding;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class RentalTest {
+
+    @Test
+    public void getRentCostNew() {
+        Movie movie = new MovieNew("The Revelant");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "The Revelant" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)